src/protocols/rdp/keymap.h

```diff
@@ -103,6 +103,24 @@ struct guac_rdp_keymap {
      */
     const UINT32 freerdp_keyboard_layout;
 
+    /**
+     * FreeRDP keyboard type definition.
+     * See https://docs.microsoft.com/ja-jp/windows/win32/api/winuser/nf-winuser-getkeyboardtype
```

Comment:
Please point to the English documentation link rather than the Japanese version.

Reply:
I will replace to English page URL.